sdk/formrecognizer/ai-form-recognizer/package.json

```diff
@@ -121,12 +124,12 @@
     "mocha": "^7.1.1",
     "mocha-junit-reporter": "^1.18.0",
     "nyc": "^14.0.0",
-    "prettier": "^1.16.4",
+    "prettier": "^2.4.0",
     "rimraf": "^3.0.0",
     "rollup": "^1.16.3",
     "sinon": "^9.0.2",
     "source-map-support": "^0.5.9",
-    "typescript": "~4.2.0",
+    "typescript": "~4.4.0",
```

Comment:
TypeScript >= 4.3.0 is _required_ to compile this package, as I am using generic constraint refinements internally.